src/js/components/deployments/deployment-report/phaseprogress.js

```diff
@@ -104,7 +104,7 @@ const ProgressChart = ({ deployment = {}, showDetails, style }) => {
     <>
       {!showDetails && (
         <>
-          Phase 1: {Math.round((device_count / max_devices) * 100)}% ({device_count} {pluralize('device', device_count)})
+          Phase 1: {Math.round((device_count / totalDeviceCount || 0) * 100)}% ({device_count} {pluralize('device', device_count)})
```

Comment:
Maybe you wanted:
```suggestion
          Phase 1: {Math.round((device_count / (totalDeviceCount || 1)) * 100)}% ({device_count} {pluralize('device', device_count)})
```

?

Reply:
while that would lead to the same, what I actually wanted to write was `((device_count / totalDeviceCount) || 0)`, to drive home the point that the entire expression should evaluate to _0_ - but `prettier` taught me that the calculation will evaluate before the boolean expression anyway... so to not mislead my future self I would like to point the counts to 0 where possible...